akka-docs/src/main/paradox/typed/from-classic.md

```diff
@@ -388,7 +388,7 @@ Links to reference documentation:
 
 The correspondence of the classic `PersistentActor` is @scala[`akka.persistence.typed.scaladsl.EventSourcedBehavior`]@java[`akka.persistence.typed.javadsl.EventSourcedBehavior`].
 
-The Typed API is much more guided to facilitate event sourcing best practises. It also has tighter integration with
+The Typed API is much more guided to facilitate Event Sourcing best practices. It also has tighter integration with
```

Comment:
As I was modifying this line, I saw that "practises" uses the English spelling and, AFAIR, Akka uses the American one.